backend/tests/Functional/Controller/PrivacyControllerTest.php

```
@@ -136,4 +137,22 @@ public function testLastAdministratorCannotDeleteOwnAccount(): void
+     * An export is a file that leaves the server, so a provider token has no
+     * business in it. Nothing about the token is the user's personal data in
+     * any sense an export serves — it is a credential for somebody else's API.
+     */
+    public function testTheExportDoesNotCarryAMetadataProviderToken(): void
+    {
+        $this->loginAs(UserFactory::createOne()->object());
+        $this->putJson('/api/me/metadata-credentials', ['metronToken' => 'personal-metron-token']);
+
+        $this->getJson('/api/privacy/export');
+
+        self::assertResponseIsSuccessful();
```

PHPStan (2.2.7)

[error] Call to an undefined static method App\Tests\Functional\Controller\PrivacyControllerTest::assertResponseIsSuccessful().


(staticMethod.notFound)